src/docfx/config/ops/DocsEnvironment.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#nullable enable
+
+namespace Microsoft.Docs.Build
+{
+    public enum DocsEnvironment
+    {
+        Production,
+        Sandbox,
+        @Internal,
```

Comment:
remove `@`?

Reply:
Sure. My original thoughts is keeping this same with paramter name. However seems no necessity.